src/builder/Actions.sol

```diff
@@ -245,6 +256,11 @@ library Actions {
         address paymentToken;
         string paymentTokenSymbol;
         uint256 paymentMaxCost;
+        // The secret used to generate the hash chain for a replayable operation. For non-replayable
+        // operations, the `nonce` will be the `nonceSecret` (the hash chain has a length of 1)
+        bytes32 nonceSecret;
+        // The number of times an operation can be replayed. For non-replayable operations, this will be 0
+        uint256 replayCount;
```

Comment:
On the backend, we've been more keen to use "totalPlays" which tends to map a little better since it doesn't have a discontinuity at 0.

Reply:
Adjusted to `totalPlays` to be consistent. Both are fine imo.